ui/tests/acceptance/secrets/backend/configuration/edit-test.js

```diff
@@ -1,47 +0,0 @@
-/**
- * Copyright (c) HashiCorp, Inc.
- * SPDX-License-Identifier: BUSL-1.1
- */
-
-import { click } from '@ember/test-helpers';
-import { module, test } from 'qunit';
-import { setupApplicationTest } from 'ember-qunit';
-import { v4 as uuidv4 } from 'uuid';
-import { SECRET_ENGINE_SELECTORS as SES } from 'vault/tests/helpers/secret-engine/secret-engine-selectors';
-import { runCmd } from 'vault/tests/helpers/commands';
-import authPage from 'vault/tests/pages/auth';
-import enablePage from 'vault/tests/pages/settings/mount-secret-backend';
-import { create } from 'ember-cli-page-object';
-import fm from 'vault/tests/pages/components/flash-message';
-const flashMessage = create(fm);
-
-module('Acceptance | secrets configuration | edit', function (hooks) {
-  setupApplicationTest(hooks);
-
-  hooks.beforeEach(function () {
-    this.uid = uuidv4();
-    return authPage.login();
-  });
-
-  test('it configures ssh ca', async function (assert) {
```

Comment:
Can we move this test to the new ssh-configuration-test (or am I blind and it's already there?)